src/app/Fake.Core.Trace/TraceListener.fs

```diff
@@ -243,11 +243,12 @@ type ConsoleTraceListener(importantMessagesToStdErr, colorMap, ansiColor) =
             | TraceData.LogMessage(text, newLine) | TraceData.TraceMessage(text, newLine) ->
                 write false color newLine text
             | TraceData.OpenTag(KnownTags.Target _ as tag, description) ->
-                write false color true (sprintf "Starting %s '%s'" tag.Type tag.Name)
-                if not (isNull description) then
-                    let msg = TraceData.TraceMessage("", true)
-                    let color2 = colorMap msg
-                    write false color2 true description
+                let msg = TraceData.TraceMessage("", true)
+                let color2 = colorMap msg
+                write false color2 false (sprintf "Starting %s '%s'" tag.Type tag.Name)
+                if not (String.IsNullOrWhiteSpace description) then
```

Comment:
why not leave `isNull`?

Note that the current solution has a small problem when using `--parallel`. The "newline` character might be printed after another target description (ie you have something like `Starting T1 T1Starting T2 T2: descriptionT1: description T2` followed by two newline characters). It's probably not a huge deal, but if we use the same color anyway we should probably create the string and use a single write with newline character here.